test/test_pynisher.py

```diff
@@ -296,24 +316,25 @@ def test_busy_in_C_library(self):
         p = psutil.Process()
         self.assertEqual(len(p.children(recursive=True)), expected_children[context])
         self.assertTrue(duration <= 2.1)
-        self.assertEqual(wrapped_function.exitcode, -15)
+        if context == 'forkserver':
```

Comment:
Also, for forkserver we get a 255 exit code with 3.6 python. This is the result of an exit signal when we go out of the 0-255 range.